library/core/src/sync/atomic.rs

```diff
@@ -110,6 +110,8 @@ use self::Ordering::*;
 use crate::cell::UnsafeCell;
 use crate::fmt;
 use crate::intrinsics;
+
+#[allow(unused_imports)]
 use crate::mem::align_of;
```

Comment:
Can you move the import into `from_mut` instead. Try to avoid disabling warnings like this since nobody will notice if it eventually becomes truly unused int he future.

Reply:
Done!